### DISCUSSION TITLE
Block visibility feature

### DISCUSSION THREAD

**Discussion by tnchuntic** [WordPress/gutenberg#48465] [Ideas]
I am sure that I am not the only one thinking about this  feature  "Block visibility".

Allows to control block visibility via:
- specific page, post, cpt
- specific user role
- specific term

**Comment by carlomanf** [comment]


**Reply by tnchuntic** [reply]
@carlomanf yes definitely previously jetpack have this functionality in widgets visibility but not sure why its not refactored to work with blocks because widgets are already removed by core wp. 

Also I know from experience that widget visibility works with widget blocks but not content/fse blocks so in summary code refactoring will not that much to have it working with content/fse blocks :).

**Reply by carlomanf** [reply]
My comment may have been poorly worded. I'm not speculating on the implementation, just supporting the idea in principle.

**Reply by tnchuntic** [reply]
No worries at all :)
